app/Http/Controllers/EnableTfaViaSmsController.php

```diff
@@ -28,11 +27,12 @@ public function enableTfaViaSms(Request $request, Client $client, Otp $otp)
 
             $key = $otp->totp($user->totpSecret);
 
-            $client->messages->create($user->phoneNumber,
-              [
-                  "from" => env('TWILIO_PHONE_NUMBER'),
-                  "body" => 'Use this code to log in: ' . $key
-              ]);
+            $client->messages->create(
+                $user->phoneNumber,
+                  [
+                      "from" => env('TWILIO_PHONE_NUMBER'),
+                      "body" => 'Use this code to log in: ' . $key
+                  ]);
```

Comment:
You should close the array at the same level as the opening bracket. And the parenthesis in a new line at the same level as `$client`